src/ResourceManager/Reservations/Commands.Reservations.Test/ScenarioTests/ReservationOrderTests.ps1

```diff
@@ -0,0 +1,54 @@
+# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+Get a single reservation
+#>
+function Test-GetReservationOrder
+{
+	$type = "Microsoft.Capacity/reservationOrders"
```

Comment:
Per the comment, please file an issue to add detailed descriptions to each test about how to create the necessary artifacts to run the test, and what values to change.  Please link the bug to the PR, and you should fix in the next sprint.

Reply:
https://github.com/Azure/azure-powershell/issues/4946